src/Init/Data/String/Basic.lean

```diff
@@ -59,9 +77,14 @@ def utf8GetAux : List Char → Pos → Pos → Char
   | c::cs, i, p => if i = p then c else utf8GetAux cs (i + c) p
 
 /--
-  Return character at position `p`. If `p` is not a valid position
-  returns `(default : Char)`.
-  See `utf8GetAux` for the reference implementation.
+Returns the character at position `p` of a string.
+
+If `p` is not a valid position, returns `(default : Char)`.
```

Comment:
```suggestion
Returns the character at position `p` of a string. If `p` is not a valid position, returns `(default : Char)`.
```
I think this makes sense as a single paragraph, because both sentences describe the return value.